### ISSUE TITLE
Enhancement for hasOne dropdown captions

### ISSUE THREAD

**Issue by mkrecek234** [atk4/ui#1759]
Steps to reproduce:
- You have a hasOne "Timezone_id" relation (which contains id and name) and a form which adds a Control for that with `$form->addControl('timezone_id')`;
- The form will include the Dropdown and caption will be "Timezone Id".

Expected behaviour:
- For hasOne fields where a Dropdown is added by default, the caption should remove the "_id" component and read "Timezone" only

![image](https://user-images.githubusercontent.com/28671486/157214734-06357d36-d53b-4e87-adff-67efd49ee26f.png)

**Comment by mvorisek** [comment]
feature request may be implemented here: https://github.com/atk4/ui/blob/1f3fb2f9cfe5471d87633944d4120cdeeec886ce/src/Form/AbstractLayout.php#L42 , not sure if we can impl. it in atk4/data directly, need repro code to check further, please submit

**Comment by PhilippGrashoff** [comment]
Hi @mkrecek234,

you can define a custom caption within the hasOne definition:
```
$this->hasOne('timezone_id', ['caption' => 'Timezone']);
```

**Comment by mvorisek** [comment]
@PhilippGrashoff yes, this is a feature request to do that automatically and it makes sense to me if 1:1

but I need a simple code to repro, constructing these repro codes by myself takes a lot of time (and in some cases the repro circumstances are hard to figure out)
